drivers/sensor/tdk/mpu6050/Kconfig

```diff
@@ -69,4 +69,11 @@ config MPU6050_GYRO_FS
 	  An X value for the config represents a range of +/- X degrees/second.
 	  Valid values are 250, 500, 1000, 2000.
 
+config MPU6050_SMPLRT_DIV
+	int "Sensor sample rate division factor"
+	default 0
+	help
+	  The base sample rate is 8kHz for the gyroscope and 1kHz for the accelerometer.
+	  A division factor of X sets the sample rate to: baserate / (1+X)
+	  Valid values are 0 through 255
```

Comment:
This should be a devicetree property

Reply:
I looked at the "Devicetree vs Kconfig" section in the documentation and I think you're referring to: "Use devicetree to describe hardware and its boot-time configuration"?

With this change I went with consistency. All the other register settings for the MPU6050 sensor are set in Kconfig. Other similar sensors such as the LSM6DSL and LIS3MDL are also handled this way. 

What is the desired outcome here:
1) one new devicetree setting, old settings remain in Kconfig (inconsistencies within MPU6050 driver)
2) all MPU6050 hardware settings are moved to DT (inconsistencies between sensor drivers)

I don't intend to do any work on other sensor drivers at this time.

Reply:
It's a bit more than "Use devicetree to describe hardware and its boot-time configuration" Kconfigs will affect every instance of the sensor. So if you have 2+ instances of this sensor they have to share the same kconfig values. In devicetree you have per-instance control.

Reply:
> What is the desired outcome here:
> 
> 1. one new devicetree setting, old settings remain in Kconfig (inconsistencies within MPU6050 driver)
> 2. all MPU6050 hardware settings are moved to DT (inconsistencies between sensor drivers)

Let's move all Build-time config settings to device-tree (No. 2), please. See how this was resolved similarly in the past (https://github.com/zephyrproject-rtos/zephyr/pull/76790#issuecomment-2275029035)

Reply:
> Kconfigs will affect every instance of the sensor
True, I had not considered this! Thank you for clarifying.

> Let's move all Build-time config settings to device-tree
Ok!